Example/JustTrack/Events.plist

```diff
@@ -24,7 +24,9 @@
 					<key>objectPayloadKeys</key>
 					<array>
 						<string>item_name</string>
-						<string>item_number</string>
+						<string>item_number_int</string>
+						<string>Item_price_double</string>
```

Comment:
```suggestion
						<string>item_price_double</string>
```

Reply:
This was intentional, proving that the lowercase title works. 

But I agree it looks messy, will revert